src/Integration.Vsix/Settings/SonarLintSettings.cs

```diff
@@ -108,12 +107,6 @@ public bool ShowServerNuGetTrustWarning
             set { this.SetValue(nameof(ShowServerNuGetTrustWarning), value); }
         }
 
-        public bool IsAnonymousDataShared
-        {
-            get { return this.GetValueOrDefault(nameof(IsAnonymousDataShared), true); }
-            set { this.SetValue(nameof(IsAnonymousDataShared), value); }
```

Comment:
I would some day add `[CallerMemberName]` in SetValue/GetValueOrDefault but perhaps not immediately...